update.sh

```diff
@@ -3,6 +3,16 @@
 KERNEL_VERSION_RPI1=3.18.0-trunk-rpi
 KERNEL_VERSION_RPI2=3.18.0-trunk-rpi2
 
+RASPBIAN_ARCHIVE_KEY_LOCATION="https://archive.raspbian.org/"
+RASPBIAN_ARCHIVE_KEY_NAME="raspbian.public.key"
+RASPBIAN_ARCHIVE_KEY_URL="${RASPBIAN_ARCHIVE_KEY_LOCATION}${RASPBIAN_ARCHIVE_KEY_NAME}"
```

Comment:
could you add a slash in between the variables? and remove the one at the end of RASPBIAN_ARCHIVE_KEY_LOCATION
it makes it more readable, and no error will happen if there is a slash at the end of RASPBIAN_ARCHIVE_KEY_LOCATION (I think at least)

the same goes for line 13